src/lurk/eval.rs

```diff
@@ -1985,4 +1988,17 @@ mod test {
         assert_ingress_egress_correctness("(+ 1 2)");
         assert_ingress_egress_correctness("(a 'b c)");
     }
+
+    #[test]
+    fn test_strum() {
+        assert_eq!(2, ReservedTag::iter().count());
```

Comment:
nit: I feel like it would make more sense to derive `EnumCount` for `ReservedTag` and use `ReservedTag::COUNT`